### PR TITLE
Add request URL info to the logger

### PR DESCRIPTION
**Pull request by samwilson**
Bug: T219233

### CODE REVIEW

File: Logger/WebRequestProcessor.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ */
+
+declare(strict_types=1);
+
+namespace Wikimedia\ToolforgeBundle\Logger;
+
+use Symfony\Component\HttpFoundation\RequestStack;
+
+/**
+ * WebProcessorMonolog extends information included in error reporting.
```
Comment (MusikAnimal) [inline Logger/WebRequestProcessor.php]: Should be `WebRequestProcessor `. This was my mistake in Event Metrics. Same with lines 3 and 21, though the "this file contains only..." comment block seems unnecessary to begin with (not sure where I picked up that habit).

Or you could just rename the file to `WebProcessorMonolog`. Doesn't matter :)

**Review by MusikAnimal — CHANGES_REQUESTED** [review]
One itsy bitsy request.

I'm guessing if a client wanted to add additional information, they could extend `WebRequestProcessor` and override `__invoke` and do a `parent::__invoke`?

File: Logger/WebRequestProcessor.php (hunk 1 above)
Comment (samwilson) [reply Logger/WebRequestProcessor.php]: Oops! Fixed. I got rid of the whole constructor docblock because it wasn't saying anything else.

**Comment by samwilson** [comment]
> I'm guessing if a client wanted to add additional information, they could extend WebRequestProcessor and override __invoke and do a parent::__invoke?

Yep, or they could create a separate processor couldn't they?

**Review by MusikAnimal — APPROVED** [review]
> or they could create a separate processor couldn't they?

I dunno, probably! As long as it's possible one way or another, because at some point I'll be adding the bundle to XTools :)

Looks good to go.

**Pull request merged**